### PR TITLE
node locally updates on args

### PR DESCRIPTION
**Pull request by ys3244**
updates on args to make backends server worked

### CODE REVIEW

File: accern_xyme/accern_xyme.py [hunk 1]
```diff
@@ -1461,6 +1462,9 @@ def update_info(self, node_info: NodeInfo) -> None:
     def get_pipeline(self) -> PipelineHandle:
         return self._pipeline
 
+    def get_node(self) -> 'NodeHandle':
```
Comment (jckhang) [inline accern_xyme/accern_xyme.py:1465]: remove

File: accern_xyme/accern_xyme.py [hunk 2]
```diff
@@ -1451,7 +1451,8 @@ def update_info(self, node_info: NodeInfo) -> None:
                 self._client,
                 value,
                 is_full=False,
-                pipeline=self.get_pipeline())
+                pipeline=self.get_pipeline(),
+                node=self.get_node())
```
Comment (jckhang) [inline accern_xyme/accern_xyme.py:1455]: self

**Review by jckhang — CHANGES_REQUESTED** [review]
some change...node handle does not need get_node, just use self

**Review by JosuaKrause — APPROVED** [review]
lgtm

**Comment by JosuaKrause** [comment]
solved by https://github.com/Accern/accern-xyme/pull/47

**Pull request closed without merging**